enterprise/setup/kubernetes_amazon_eks.mdx

```diff
@@ -0,0 +1,430 @@
+---
+title: 'Enterprise Edition: Initial Setup (Kubernetes on Amazon EKS)'
+backlink_href: /docs/enterprise
+backlink_title: 'pganalyze Enterprise Edition'
+---
+
+import ToC from '../../components/Toc'
+
+These are the installation instructions for [pganalyze Enterprise Edition](/enterprise-postgres-monitoring), targeted for a Kubernetes cluster managed by Amazon EKS in your own AWS account.
+
+<ToC items={props.toc} />
+
+## Installation steps
+
+### Pre-requisites
+
+* Provision an Amazon EKS cluster in your AWS account
+* Install the [AWS Load balancer controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html) on your EKS cluster
+* Provision a Postgres database (e.g. with Amazon RDS or Amazon Aurora) to store pganalyze statistics information
+* AWS CLI (`aws`) installed on your machine, and logged in with administrator permissions on your AWS account
+* Kubernetes CLI (`kubectl`) installed on your machine, and logged into the Kubernetes cluster with full permissions
+
+### Step 1: Re-publish the Docker image to your private container registry
+
+Perform the next steps on a machine that has full internet connectivity, e.g your personal machine, or a VM, with no outbound access restrictions.
+
+The goal of this first step is to publish the pganalyze software to your private AWS Container Registry, which can be accessed from your Amazon EKS cluster.
+
+First, login with your license information, shared with you by the pganalyze team:
```

Comment:
I'll add some "login" / "log in" and "setup" / "set up" nitpicking, feel free to ignore but here it goes: this should be "log in".